src/app/shared/util/metrics.util.ts

```diff
@@ -1,29 +1,22 @@
-import { EpisodeModel, RouterModel,
-  PodcastMetricsModel, EpisodeMetricsModel, getMetricsProperty,
+import { RouterParams, PodcastMetricsModel, EpisodeMetricsModel, getMetricsProperty,
   IntervalModel, INTERVAL_HOURLY, INTERVAL_DAILY, INTERVAL_WEEKLY, INTERVAL_MONTHLY } from '../../ngrx';
 import * as dateUtil from './date/date.util';
 
-export const filterPodcastEpisodePage = (filter: RouterModel, episodes: EpisodeModel[]) => {
-  if (filter && filter.podcastSeriesId && filter.page && episodes) {
-    return episodes.filter(episode => episode.seriesId === filter.podcastSeriesId && episode.page === filter.page);
-  }
-};
-
 export const findPodcastMetrics =
```

Comment:
honestly this findPodcastMetrics function should just go with the podcast.selector it's used in.